File: documentation/user/en/query/ordering/examples/natural/attribute-natural-localized.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("cs"))
        	),
        	OrderBy(
        		AttributeNatural("name", Asc)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("name")
        		)
        	)
        )
	)
);

File: documentation/user/en/query/ordering/examples/comparable/attribute-natural-localized.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityLocaleEquals(Locale.forLanguageTag("cs"))
				),
				orderBy(
					attributeNatural("name", ASC)
				),
				require(
					entityFetch(
						attributeContent("name")
					)
				)
			)
		);
	}
);
